T2CC_APP/t2cc/app/src/main/res/layout/appbar.xml

```diff
@@ -14,7 +14,7 @@
         android:layout_marginLeft="5dp"
         android:layout_width="48dp"
         android:layout_height="50dp"
-        android:layout_marginRight="215dp"
+        android:layout_marginRight="200dp"
```

Comment:
Can we give these a bit more room. I want the messages farther than the log out. It's the main function of our app and I think right now it's ways too close to the sign out button. So maybe we need to switch the initials and sign out fields. That way a slip of a finger when trying to get your messages wont log you out